File: transaction-bff/src/main/java/br/com/felipe/gadelha/coffeandit/transactionbff/domain/entity/TransactionStatus.java
package br.com.felipe.gadelha.coffeandit.transactionbff.domain.entity;

import io.swagger.v3.oas.annotations.media.Schema;

@Schema
public enum TransactionStatus {
    ANALYSED,
    NOT_ANALYSED,
    IN_HUMAN_ANALYSIS,
    IN_SUSPECTED_FRAUD,
    CONFIRMED_RISK
}

File: limit-ms/src/main/kotlin/br/com/felipe/gadelha/coffeandit/limitms/domain/entity/TransactionStatus.kt
package br.com.felipe.gadelha.coffeandit.limitms.domain.entity

enum class TransactionStatus {
    ANALYSED,
    NOT_ANALYSED,
    IN_HUMAN_ANALYSIS,
    IN_SUSPECTED_FRAUD,
    CONFIRMED_RISK
}

//enum class GatewayType {
//    PAYPAL { override fun getUri(): String { return "paypal.com?buyerId=" } },
//    PAGSEGURO { override fun getUri(): String { return "pagseguro.com?returnId=" } };
//    abstract fun getUri(): String
//}